docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+
+Mono
+----
+
+[Mono](http://www.mono-project.com/docs/about-mono/) is the original open source and cross-platform implementation of .NET, from the community [Mono Project](http://mono-project.com). It is now sponsored by Microsoft. It can be thought of as an open and cross-platform version of the .NET Framework. It's APIs follow the progress of the .NET Framework, not .NET Core.
+
+There are several components that make up Mono:
+
+**C# Compiler** - Mono’s C# compiler is feature complete for C# 6.
+
+**Mono Runtime** - The runtime implements the ECMA Common Language Infrastructure (CLI). The runtime provides a Just-in-Time (JIT) compiler, an Ahead-of-Time compiler (AOT), a library loader, the garbage collector, a threading system and interoperability functionality.
+
+**Base Class Library** - The Mono platform provides a comprehensive set of classes that provide a solid foundation to build applications on. These classes are compatible with Microsoft’s .Net Framework classes.
+
+**Mono Class Library** - Mono also provides many classes that go above and beyond the Base Class Library provided by the .NET Framework. These provide additional functionality that are useful, especially in building Linux applications. Some examples are classes for Gtk+, Zip files, LDAP, OpenGL, Cairo, POSIX, etc.
+
+Unity
+-----
+
+[Unity](http://unity3d.com) is a [popular](http://unity3d.com/public-relations) cross-platform 2D + 3D game engine that uses C# for scripting game play and logic. Unity has several graphics API back-ends including DirectX for Windows and OpenGL for any Operating System (OS) that supports it. A major benefit of Unity is its broad platform support. You can build a single game and publish it to the more than [25 platforms that Unity supports](http://unity3d.com/unity/multiplatform). These platforms include all the popular targets in desktop, console, mobile, VR and AR gaming.
+
+The [Made with Unity](http://madewith.unity.com) site showcases published Unity games. These games demonstrate the breadth of possibility using Unity.
+
+Unity has traditionally used the Mono runtime to execute "C# scripts". More recently, Unity has been adopting its [IL2CPP](http://blogs.unity3d.com/2014/05/20/the-future-of-scripting-in-unity/) technology.
+
+.NET Native
+-----------
+
+.NET Native is a set of native build tools for .NET Core. .NET Native is an Ahead-of-Time (AOT) toolchain that produces native applications by compiling IL byte code to native machine code, . This means that the resulting binary is what the OS executes; there is no JIT-ing, no runtime compilation. This leads to better startup performance, as well as some security benefits.
```

Comment:
If we are introducing .NET Native as "a set of native build tools for .NET Core." and we're following the "order of introduction" organizing principle, this should come after .NET Core.